src/libstd/io/mod.rs

```diff
@@ -1376,6 +1377,67 @@ pub trait Write {
         Ok(())
     }
 
+    /// Attempts to write an multiple buffers into this writer.
+    ///
+    /// This method will continuously call [`write_vectored`] until there is no
+    /// more data to be written or an error of non-[`ErrorKind::Interrupted`]
+    /// kind is returned. This method will not return until all buffers have
+    /// been successfully written or such an error occurs. The first error that
+    /// is not of [`ErrorKind::Interrupted`] kind generated from this method
+    /// will be returned.
+    ///
+    /// If the buffer contains no data, this will never call [`write_vectored`].
+    ///
+    /// # Notes
+    ///
+    /// Different to `io::Write::write_vectored` this takes a *mutable*
+    /// reference to a slice of `IoSlice`s, not a non-mutable reference, because
+    /// we need to modify the slice to keep track of the bytes already written.
+    ///
+    /// Once this function returns the contents of `bufs` is undefined, not
```

Comment:
This can just say that "the contents of `bufs` is unspecified".

Reply:
Done.